### PR TITLE
Update references to covid policy across the site to allow for mask-optional events

### PR DESCRIPTION
**Pull request by harrislapiroff**
Do not merge until we are also ready to publish our blog post about the updates

### CODE REVIEW

**Comment by harrislapiroff** [comment]
Ah, fair enough. Okay, I'll adjust the language to reflect that we have both kinds of dances without specifics about the schedule yet. Might be good to keep this commit in mind to revert to assuming all goes to plan.

**Comment by harrislapiroff** [comment]
Okay, done that @jeffkaufman

**Comment by harrislapiroff** [comment]
Also made it a draft so I don't make an error and merge it too soon 😅

File: dances.js [hunk 1]
```diff
@@ -744,11 +744,15 @@ e = [
   },
   {
     "date": [2023, 6, 18],
+    "caller": "Hannah Chamberlain",
     "band": "Cojiro: Jess Newman, Mark Price",
+    "html": "This dance is <a href=\"/accessibility\"><strong>mask-required</strong></a>"
```
Comment (jeffkaufman) [inline dances.js:749]: nit: you can just write `<a href=/accessibility>`.  Or `<a href='/accessibility'>` if you really want quotes.

File: accessibility.html [hunk 2]
```diff
@@ -13,39 +13,34 @@ <h1>Accessibility</h1>
...
 
-As of 2023-05-21 we've resumed our potlucks.  People are welcome to
-eat outside or inside.  If you're eating inside, please wear your mask
-when you're not eating.
+<li><p>Please notify us after if you learn you may have been
+infectious at the dance.  Without giving your name we’ll let the
+community know by posting on the website and emailing our
+<a href="https://www.bidadance.org/join">covid notification list</a>.
+
+</ul>
```
Comment (jeffkaufman) [inline accessibility.html:43]: I think we should probably say we no longer require vaccination, so people looking for a vaccination requirement know they can stop searching?

Comment (harrislapiroff) [reply accessibility.html:43]: Oh right, we also did say we were going to recommend people stay up to date

File: dances.js (hunk 1 above)
Comment (harrislapiroff) [reply dances.js]: Done!

File: accessibility.html (hunk 2 above)
Comment (harrislapiroff) [reply accessibility.html:46]: Done!

**Pull request merged**